ORM/ORMFundamentals/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ORMFundamentals.Data;
+using ORMFundamentals.Entities;
+using System.Linq.Expressions;
+
+namespace ORMFundamentals.Repositories;
+
+public class OrderRepository : IOrderRepository
+{
+    private readonly AppDbContext _context;
+
+    public OrderRepository(AppDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<IEnumerable<Order>> GetAllOrders()
+    {
+        return await _context.Orders.ToListAsync();
+    }
+
+    public async Task<Order> GetOrderById(int id)
+    {
+        return await _context.Orders.FindAsync(id);
+    }
+
+    public async Task AddOrder(Order order)
+    {
+        _context.Orders.Add(order);
+        await _context.SaveChangesAsync();
```

Comment:
Just a hint, no need to add changes: usually, it's a good idea to use [_Unit Of Work_](https://learn.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-the-repository-and-unit-of-work-patterns-in-an-asp-net-mvc-application) design pattern with _Repository_. The simple answer - fluent ability to manage multiple DB operations via single transaction (`SaveChanges` method with its overloaded versions is making transaction). Keep than in mind for the future, it might be also useful if you attempt to modify your entity in multiple code parts, that could lead to concurrency exception for EF.
You can see my implementation [here](https://github.com/dotnet-fizzyy/Diploma/blob/V2/WebAPI/WebAPI/Infrastructure/Postgres/Repository/UnitOfWork.cs#L7) for the future.